src/Compilers/Core/Portable/Text/SourceText.cs

```diff
@@ -12,7 +12,6 @@
 using System.Linq;
 using System.Text;
 using System.Threading;
-using Microsoft.CodeAnalysis.Debugging;
```

Comment:
Consider reverting changes in this file.

Reply:
The import is no longer needed as the namespace of SourceHashAlgorithms changed to Microsoft.CodeAnalysis.Text.